contracts/src/v0.8/dev/interfaces/KeeperRegistryInterfaceDev.sol

```diff
@@ -76,6 +76,7 @@ struct Upkeep {
 }
 
 interface KeeperRegistryBaseInterface {
+  // split this to a new interface for keeper registrar
```

Comment:
We can do this later probably since it's needed for registrar

Reply:
Yes